### PR TITLE
Update warning to caution

### PR DESCRIPTION
**Pull request by FengUyu**

### CODE REVIEW

File: docs-2.0/3.ngql-guide/6.functions-and-expressions/5.case-expressions.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ WHEN <value> THEN <result>
 END
 ```
 
-!!! waring "警告"
+!!! caution "警告"
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/6.functions-and-expressions/5.case-expressions.md:23]: No need to customize the title. Same as the following modifications.

File: docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md [hunk 2]
```diff
@@ -12,7 +12,7 @@
 
 ## 使用索引必读
 
-!!! warning "不要任意在生产环境中使用索引，除非您很清楚使用索引对业务的影响。"
+!!! caution "不要任意在生产环境中使用索引，除非您很清楚使用索引对业务的影响。"
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md:15]: !!! caution
    不要轻易在生产环境中使用索引，除非您很清楚使用索引对业务的影响。索引会导致写性能下降90%甚至更多。索引并不用于查询加速。只用于根据属性定位到点或边，或者统计点边数量。

File: docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md [hunk 3]
```diff
@@ -26,7 +26,7 @@
 
 4. 使用[LOOKUP](../7.general-query-statements/5.lookup.md)或[MATCH](../7.general-query-statements/2.match.md)语句查询数据。您不需要指定使用哪个索引，Nebula Graph会自动计算。
 
-!!! warning "如果先创建索引再导入数据，会因为写性能的下降导致导入速度极慢。"
+!!! caution "如果先创建索引再导入数据，会因为写性能的下降导致导入速度极慢。"
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md:29]: !!! caution
        如果先创建索引再导入数据，会因为写性能的下降导致导入速度极慢。

File: docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md [hunk 4]
```diff
@@ -56,7 +56,7 @@ CREATE {TAG | EDGE} INDEX [IF NOT EXISTS] <index_name> ON {<tag_name> | <edge_na
   
   + 为标签或边类型本身创建索引时，请忽略`prop_name_list`。
 
-!!! warning "长索引会降低Storage服务的扫描性能，以及占用更多内存。建议您将索引长度设置为和要被索引的最长字符串相同。索引长度最长为255，超过部分会被截断。"
+!!! caution "长索引会降低Storage服务的扫描性能，以及占用更多内存。建议您将索引长度设置为和要被索引的最长字符串相同。索引长度最长为255，超过部分会被截断。"
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md:59]: Same as above.

File: docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md [hunk 5]
```diff
@@ -98,7 +98,7 @@ nebula> CREATE EDGE INDEX follow_index_0 on follow(degree);
 
 通一个便签或者边类型中的多个属性上的索引称为复合索引。
 
-!!! warning "不支持跨标签或边类型创建复合索引。"
+!!! caution "不支持跨标签或边类型创建复合索引。"
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md:101]: Same as above.

File: docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md [hunk 6]
```diff
@@ -34,7 +34,7 @@ nGQL不支持遍历所有点和边，例如`MATCH (v) RETURN v`。
 
 请确保`MATCH`语句有至少一个索引可用。如果您需要创建索引，但是已经有相关的点、边或属性，您必须在创建索引后重建索引，索引才能生效。
 
-!!! waring "警告"
+!!! caution "警告"
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md:37]: Same as above.

File: docs-2.0/3.ngql-guide/8.clauses-and-options/ttl-options.md [hunk 7]
```diff
@@ -95,6 +95,6 @@ nebula> INSERT VERTEX t2(a, b, c) values "102":(1612778164674, 30, "Hello");
     ```ngql
     nebula> ALTER TAG t1 ttl_duration = 0;
     ```
-  !!! waring "警告"
+  !!! caution "警告"
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/ttl-options.md:98]: Same as above.

File: docs-2.0/3.ngql-guide/9.space-statements/1.create-space.md [hunk 8]
```diff
@@ -39,7 +39,7 @@ CREATE SPACE [IF NOT EXISTS] <graph_space_name>
 
     指定每个分片的副本数量。建议在生产环境中设置为3，在测试环境中设置为1。由于需要基于多数表决，副本数量必须是**奇数**。默认值为1。
 
-  !!! waring "注意"
+  !!! caution "注意"
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/9.space-statements/1.create-space.md:42]: Same as above.

File: docs-2.0/3.ngql-guide/9.space-statements/5.drop-space.md [hunk 9]
```diff
@@ -16,6 +16,6 @@ DROP SPACE [IF EXISTS] <graph_space_name>;
 
 `DROP SPACE`语句不会立刻删除硬盘上对应图空间的目录和文件，请使用`USE`语句指定其他任意图空间，然后执行`SUBMIT JOB COMPACT`。
 
-!!! waring "警告"
+!!! caution "警告"
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/9.space-statements/5.drop-space.md:19]: Same as above.

**Review by randomJoe211 — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Pull request merged**